<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QGridLayout" name="gridLayout_MinValues">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_Saturation_min">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_hue_min">
       <property name="text">
        <string>Hue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Saturation_min">
       <property name="text">
        <string>Saturation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Saturation_min">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_Hue_min">
       <property name="maximum">
        <number>179</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_Hue_min">
       <property name="maximum">
        <number>179</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_Value_min">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_Value_min">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Value_min">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <layout class="QGridLayout" name="gridLayout_MaxValues">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_Saturation_max">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Hue_max">
       <property name="text">
        <string>Hue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Saturation__max">
       <property name="text">
        <string>Saturation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Saturation_max">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_Hue_max">
       <property name="maximum">
        <number>179</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_Hue_max">
       <property name="maximum">
        <number>179</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_Value_max">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_Value_max">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Value_max">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Hue_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Hue_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Hue_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Hue_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Saturation_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Saturation_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Saturation_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Saturation_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Value_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Value_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Value_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Value_min</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Hue_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Hue_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Hue_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Hue_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Saturation_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Saturation_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Saturation_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Saturation_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Value_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Value_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Value_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Value_max</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
